### PR TITLE
Badge updates

### PR DESCRIPTION
**Pull request by ishanfdo18098**
- [x] Badges of Casual Instructors
- [x] Web Consultation Team
- [x] ACES committees

### CODE REVIEW

File: societies/aces-2023,json
Comment (NuwanJ) [inline societies/aces-2023,json:1]: `2023,json` can you replace the `,` with `.`

Comment (ishanfdo18098) [reply societies/aces-2023,json:1]: Thank you

File: badges/casual_instructors_2024.json [hunk 1]
```diff
@@ -0,0 +1,381 @@
+{
+    "title": "Casual Instructors - 2024",
+    "subtitle": "Semester 3 and 5",
```
Comment (ishanfdo18098) [inline badges/casual_instructors_2024.json:3]: What should be the correct semster range ?

File: badges/casual_instructors_2023_2024.json [hunk 2]
```diff
@@ -0,0 +1,311 @@
+{
+    "title": "Casual Instructors - 2023/2024",
+    "subtitle": "Semester 3 and 5",
```
Comment (ishanfdo18098) [inline badges/casual_instructors_2023_2024.json:3]: What should be the correct semster range ?

File: badges/casual_instructors_2024.json (hunk 1 above)
Comment (chandula00) [reply badges/casual_instructors_2024.json:3]: Semester 2, 4 and 6

**Pull request merged**